app/views/users/_form_fields.html.erb

```diff
@@ -41,6 +41,12 @@
   <%= f.select :organisation_id, organisation_options(f), organisation_select_options, { class: "chosen-select form-control", 'data-module' => 'chosen' } %>
 </p>
 
+<% if policy(User).flag_2sv? %>
+  <p class="form-group">
+    <%= f.check_box :requires_2sv %> <%= f.label :requires_2sv, "Requires 2SV" %>
```

Comment:
Thanks for the screenshot :)

Can the checkbox be moved up, so it's between role and organisation. To begin with the need for 2SV is coupled with a user's role.

The `<p class="">` wrapper should also have the class "checkbox" as per the Bootstrap 3 pattern. This should stop the label from showing in bold.